CHANGELOG.md

```diff
@@ -1,8 +1,98 @@
 # Changelog for Elixir v1.10
 
-## v1.10.0-dev
+## Support for Erlang/OTP 21+
 
-Elixir v1.10 requires Erlang/OTP 21+.
+Elixir v1.10 requires Erlang/OTP 21+, allowing Elixir to integrate with Erlang/OTP's new logger. Currently, this means that the logger level, logger metadata, as well as all log messages are now shared between Erlang and Elixir APIs.
+
+We will continue improving the relationship between the logging systems in future releases. In particular, we plan to expose all log levels and runtime filtering functionalities available in Erlang directly into Elixir in the next Elixir version.
+
+This release also adds two new guards, `is_struct/1` and `is_map_key/2`, thanks to the strict requirement on Erlang/OTP 21+.
+
+## Releases improvements
+
+Elixir v1.9 introduced releases as a mechanism to package self-contained applications. Elixir v1.10 further improves releases with bug fixes and new enhancements based on feedback we got from the community. The highlights are:
+
+  * Allow the dual boot system of releases to be disabled on environments that are boot-time sensitive, such as embedded devices
+
+  * Track and raise if compile-time configuration is set or changes at runtime (more in the next section)
+
+  * Support for easily adding extra files to releases via overlays
+
+  * Allow `RELEASE_DISTRIBUTION` to be set to `none` in order to fully disable it
+
+  * Add a built-in `:tar` step that automatically packages releases
+
+See the full CHANGELOG for more improvements.
+
+## Improvements to sort-based APIs in Enum
+
+`Enum.sort/1` in Elixir always sorts from lowest to highest. If you want to sort from highest to lowest, you need to call `Enum.sort/2` with a custom sorting function, such as `Enum.sort(collection, &>=/2)`, which is not immediately obvious to someone reading the code.
+
+To make matters worse, comparison operators, such as `<=` and `>=`, perform structural sorting, instead of a semantic one. For example, using `>=` to sort dates descendinglly won't yield the correct result. Therefore, to sort dates from more recent to oldest, one has to write `Enum.sort(dates, &(Date.compare(&1, &2) != :lt))`.
```

Comment:
```suggestion
To make matters worse, comparison operators, such as `<=` and `>=`, perform structural sorting, instead of a semantic one. For example, using `>=` to sort dates descendingly won't yield the correct result. Therefore, to sort dates from more recent to oldest, one has to write `Enum.sort(dates, &(Date.compare(&1, &2) != :lt))`.
```